docs/dev/useful_models.rst

```diff
@@ -0,0 +1,17 @@
+Commonly Used Models
+====================
+Since [this picture](//web.mit.edu/benkraft/Public/esp-models.png) may not be the most useful way to figure out what model does what you want, here’s a list of the most commonly used ones:
+* ``esp.users.models.ESPUser``: This is a proxy model over ``django.contrib.auth.User`` (the standard user account model). Proxy model basically means it attaches extra methods, but not extra fields.
```

Comment:
Add line break before the start of the bullets, otherwise they don't get displayed as bullets / there aren't line breaks between the bullets.